tests/integration/targets/module_utils_core/roles/ansibleawsmodule.client/tasks/credentials.yml

```diff
@@ -42,7 +42,7 @@
 # Tests using standard credentials from environment variables
 
 - name: 'Test basic operation using simple credentials (aws-environment)'
-  boto3_example:
+  example_module:
```

Comment:
This fails with local credentials.
`botocore.exceptions.ClientError: An error occurred (AuthFailure) when calling the DescribeImages operation: AWS was not able to validate the provided access credentials`